### DISCUSSION TITLE
トラックボールが突然反応しなくなった

### DISCUSSION THREAD

**Discussion by lokman928** [Yowkees/keyball#634] [Q&A]
こんにちは、私は DIY はんだ付けで keyball44 を使用しています。トラックボールが突然反応しなくなるという問題に直面しています (たとえば、トラックボールは正常に動作しますが、トラックボールを動かしたり変更を加えたりしていなくても、キーボールを PC に接続すると、突然反応しなくなります)物事は正常に動作しますが、トラックボールだけが応答しません）、LEDからの情報を確認しました。トラックボールの値は0、0です。

この問題を解決するには、通常はキーボールを削除して再度接続すると問題が解決しますが、解決できない場合もあります。

誰かがこの問題を解決する方法を見つけるのを手伝ってくれますか?

(by google translate)

**Comment by Yowkees** [comment]
Hi, Thank you for choosing Keyball44.

I think there is some problem in soldering. Can you show me the picture of back side of PCB. Specially ProMicro and 7P-conthru.

Thank you very much.

**Comment by lokman928** [comment]
![WhatsApp 圖片2024-08-1123 51 23_0ef28e6c](https://github.com/user-attachments/assets/2efe3b2f-1ed5-44c8-96ec-e73ea380f709)
![WhatsApp 圖片2024-08-1123 51 23_3fe0be65](https://github.com/user-attachments/assets/921ee55b-a55a-4638-9a51-52bb095f365e)
![WhatsApp 圖片2024-08-1123 51 23_700464b0](https://github.com/user-attachments/assets/bfbe56f5-7405-4370-84d4-0ab6a6d7660d)
![WhatsApp 圖片2024-08-1123 51 24_6011f203](https://github.com/user-attachments/assets/5f19f8a4-1a90-462e-a62c-1cf9059031f2)
Let me know if you need more image 🙏

**Comment by lokman928** [comment]
Hi, can I have any support for this question 🙏

**Comment by Yowkees** [comment]
Hi. I can't find problem on soldering.
Is there any change if you swap right ProMicro and left one.
If it's NO, I think Track-ball optical sensor is broken.

If you want to try new track-ball PCB, please contact me from below link,
https://shirogane-lab.net/inquiry

Thank you.
